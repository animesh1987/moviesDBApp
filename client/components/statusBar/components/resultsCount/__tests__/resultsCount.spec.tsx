import * as React from 'react';
import { create } from 'react-test-renderer';

import { ResultsCount } from '../index';

describe('Results Count Test', () => {
  const count = 1;

  it('renders correctly', () => {
    const component = create(<ResultsCount count={count}/>)
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});