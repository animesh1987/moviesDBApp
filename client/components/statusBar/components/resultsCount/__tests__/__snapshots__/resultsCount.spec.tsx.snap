// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Results Count Test renders correctly 1`] = `
<span
  className="results"
>
  1
   movies found
</span>
`;
